scripts/billing/sql/total_costs_by_sku_label.sql

```diff
@@ -0,0 +1,10 @@
+/*
+Query returns total costs by sku and all labels
+*/
+
+SELECT
+  sku.description,
```

Comment:
Alias this field

```sql
  sku.description as sku_description
```

Reply:
Aliased this field and similar ones across files